### DISCUSSION TITLE
Kotlin Multiplatform with iOS, Android and JVM Desktop targets

### DISCUSSION THREAD

**Discussion by jQrgen** [touchlab/CrashKiOS#94] [Feature request]
Using CrashKiOS crash reporting library in commonMain with iOS, Android and JVM Desktop targets crashes build because CrashKiOS does not support the JVM target.

Is there any way to get around this issue in the dependency three?

What would it take to add a JVM target?

**Comment by nanthakumarg** [comment]
same issue here too. +1

**Comment by tamimattafi** [comment]
@jQrgen @nanthakumarg 
JVM probably doesn't have a Crashlytics SDK (There's nothing in the dashboard about JVM and Desktop too)
We use our dependecies like this at the moment
```kotlin
val commonMain by getting {
    dependencies {
        // Logger
        implementation(libs.kermit)
    }
}

val mobileMain by creating {
    dependsOn(commonMain)

    dependencies {
        // Logger
        implementation(libs.crashkios)
        implementation(libs.kermit.crashlytics)
    }
}

val androidMain by getting {
    dependsOn(mobileMain)
    dependencies {
        // Android specific dependencies
    }
}

val iosMain by getting {
    dependsOn(mobileMain)
    dependencies {
        // iOS specific dependencies
    }
} 
```
Then you can create `shared/src/mobileMain` that will contain common code for mobile

**Reply by wellingtoncosta** [reply]
Hi folks, we're planning to add no-op support for JVM / JS. I just created this discussion: https://github.com/touchlab/CrashKiOS/discussions/101

**Comment by wellingtoncosta** [comment]
We released [0.10.0](https://github.com/touchlab/CrashKiOS/releases/tag/0.10.0) and it adds no-op support for JVM and JS.
